### ISSUE TITLE
Editor is duplicating special characters such as á

### ISSUE THREAD

**Issue by leogermani** [Automattic/simplenote-electron#2330]
Using the web app on Firefox and Mac OS

### Expected

When I type a letter with an accent, such as "á" I wanted it to be entered only once.

### Observed

It's duplicating

### Reproduced

1. Open the web app in Firefox
2. type "á"
3. see it duplicates

### Where did you see the bug

- System Make:
- System Model:
- OS: Mac OS
- OS version: Catalina
- Browser (if applicable): Firefox
- Browser version (if applicable):  80
- Simplenote app version:

**Comment by leogermani** [comment]
Closing in favor of #2327
